It seems to only try and download fast_float from github instead of using the
installed version.

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -1040,13 +1040,6 @@ ENDIF()
 
 include("cmake/Check-from_chars.cmake")
 IF(NOT FROM_CHARS_WORKS)
-    CPMFindPackage(NAME fast_float
-                   GIT_REPOSITORY https://github.com/fastfloat/fast_float
-                   VERSION 6.1.0
-                   EXCLUDE_FROM_ALL yes)
-    GET_TARGET_PROPERTY(fast_float_INCLUDE_DIRECTORIES
-                        FastFloat::fast_float INTERFACE_INCLUDE_DIRECTORIES)
-    INCLUDE_DIRECTORIES(${fast_float_INCLUDE_DIRECTORIES})
     ADD_DEFINITIONS(-DUSE_FAST_FLOAT)
 ENDIF()
 
